.github/workflows/test-template.yml

```diff
@@ -107,7 +107,7 @@ jobs:
   # CLIENT (functional and legacy tests without types test)
   #
   client:
-    name: Client func/legacy-notypes
+    name: Client func & legacy-notypes
```

Comment:
```suggestion
    name: Client func&legacy-notypes
```

Shorter would be better because it cuts before we can see if it's "l..." for "library" here for example
https://github.com/prisma/prisma/actions/runs/7059347363/job/19216734376?pr=22189
<img width="292" alt="Screenshot 2023-12-01 at 12 30 50" src="https://github.com/prisma/prisma/assets/1328733/3b2bb931-df02-4030-9329-33e282655b62">